src/raster/r.forcircular/r.forcircular.py

```diff
@@ -0,0 +1,2038 @@
+#!/usr/bin/env python3
+#
+############################################################################
+#
+# MODULE:      r.forcircular
+# AUTHOR(S):   Francesco Geri, Sandro Sacchelli
+# PURPOSE:     Evaluation of circular bioeconomy level of forest ecosystems
+# COPYRIGHT:   (C) 2022 by the GRASS Development Team
+#
+#              This program is free software under the GNU General Public
+#              License (>=v2). Read the file COPYING that comes with GRASS
+#              for details.
+#
+#############################################################################
+#%Module
```

Comment:
Please insert a space between # and % `# %` in all these lines (see also #693).

Reply:
done

Reply:
Done